services/app-api/resources/ssm-params.yml

```diff
@@ -0,0 +1,40 @@
+Resources:
+  CmsSpaFormFromEmail:
+    Type: AWS::SSM::Parameter
+    Properties:
+      Type: StringList
+      Description: Branch specific email address of sender for cms spa form submission emails
+      Name: !Sub /configuration/${self:custom.stage}/email/cms_spa_form_from_email
+      Value: "spa-reply@cms.hhs.gov"
+
+  CmsUserAccessFromEmail:
+    Type: AWS::SSM::Parameter
+    Properties:
+      Type: StringList
+      Description: Branch specific email address of sender for cms user access emails
+      Name: !Sub /configuration/${self:custom.stage}/email/cms_user_access_from_email
+      Value: "OneMACAccess@cms.hhs.gov"
+
+  CmsSpaFormCmsEmail:
+    Type: AWS::SSM::Parameter
+    Properties:
+      Type: StringList
+      Description: Branch specific email address of recipient for cms spa form submission emails
+      Name: !Sub /configuration/${self:custom.stage}/email/cms_spa_form_cms_email
+      Value: "OneMAC@cms.hhs.gov"
+
+  CmsSpaFormChipEmail:
+    Type: AWS::SSM::Parameter
+    Properties:
+      Type: StringList
+      Description: Branch specific email address of recipient for cms chip spa form submission emails
+      Name: !Sub /configuration/${self:custom.stage}/email/cms_spa_form_chip_email
+      Value: "OneMAC@cms.hhs.gov"
+
+  CmsSpaFormChipCcEmail:
```

Comment:
looks like this value defaults to "" in previous branches

Reply:
oh yes, this is a good point I forgot to mention in the PR. As is it would send a duplicate email to that address since it is both the default To: and CC: address, we probably dont care in environments < production and in production we will need to update the CC with the current list of individual email addresses once deployed. I couldnt figure out a way though to get cloudformation to accept an empty or null parameter value via serverless to get it setup so decided to go this route.